src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -115,11 +132,24 @@ export default {
 		},
 		trackCategory({ success }, category) {
 			if (success) this.$kvTrackEvent('loan-card', 'add-to-basket', `${category}-lending-home`);
+		},
+		closeToast() {
+			this.$refs.welcomeToastMessage.close();
```

Comment:
Ticket calls out tracking clicks, so make sure to add the tracking events listed in the ticket.

Reply:
right, ty

Reply:
Done